gender_analysis/analysis/analysis.py

```diff
@@ -34,14 +34,14 @@ def get_count_words(novel, words):
     Returns a dictionary where the keys are the elements of 'words' list
     and the values are the numbers of occurences of the elements in the novel.
     N.B.: Not case-sensitive.
-    >>> from gender_analysis import novel
+    >>> from gender_analysis import document
     >>> summary = "Hester was convicted of adultery. "
     >>> summary += "which made her very sad, and then Arthur was also sad, and everybody was "
     >>> summary += "sad and then Arthur died and it was very sad.  Sadness."
     >>> novel_metadata = {'author': 'Hawthorne, Nathaniel', 'title': 'Scarlet Letter',
     ...                   'corpus_name': 'sample_novels', 'date': '1850',
     ...                   'filename': None, 'text': summary}
-    >>> scarlett = novel.Novel(novel_metadata)
+    >>> scarlett = document.Novel(novel_metadata)
```

Comment:
this needs to be, e.g., document.Document -- and though the above variable can stay named novel_metadata, it's probably better if we call it document_metadata